generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management-model.tsx

```diff
@@ -103,7 +103,7 @@ export class UserManagementModel extends React.Component<IUserManagementModelPro
               <AvFeedback>This field is required to be at least 5 characters.</AvFeedback>
             </AvGroup>
             <AvGroup>
-              <Label check inline>
+              <Label check inline className="form-check-label">
```

Comment:
@qmonmert adding this messes up the alignment
![image](https://user-images.githubusercontent.com/1107223/33668102-0a694570-da9f-11e7-83fb-f663aa70bbb3.png)

Reply:
so i'm not sure if its a good idea

Reply:
Its needed only if your checkbox is placed before the label, here it is after the label

Reply:
With Chrome it renders like this
![capture d ecran 2017-12-06 a 16 09 16](https://user-images.githubusercontent.com/9989211/33668391-e80f3ab0-da9f-11e7-91c7-6afbd2abd09a.png)